protected/humhub/modules/content/widgets/ContentTagPicker.php

```diff
@@ -53,7 +53,10 @@ protected function findDefaults()
     {
         /* @var ContentTagActiveQuery $query */
         $query = call_user_func([$this->itemClass, 'findByContainer'], $this->contentContainer, true);
-        $query->readable()->limit($this->limit);
+        $query->limit($this->limit);
+        if (!Yii::$app->user->identity->canManageAllContent()) {
+            $query->readable();
```

Comment:
@gevorgmansuryan Hmm, it would probably be good to solve this within the `readable()` method. Or is there something against it here?